### ISSUE TITLE
suggestions for code/computer setup

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-tutorials#295]
* don't use one-letter options in a tutorial - or you will have to explain them later. if you use the full version, they are self-explanatory. i.e. instead of
```
verdi code setup -L add --computer=localhost -P arithmetic.add --remote-abs-path=/bin/bash -n
```
use
```
verdi code setup --label add --computer localhost --input-plugin arithmetic.add --remote-abs-path /bin/bash --non-interactive
```

 * typo `\bin\bash` in instructions

 * consider removing the `verdi computer show localhost` - not sure why people need to worry about this at this stage
   this will also save you the "one set of pks for one type of entity discussion"

**Comment by ltalirz** [comment]
The suggestion to avoid one-letter options applies across the board, e.g. also for `verdi group list -C` in the pw tutorial (=> `verdi group list --count`)

**Comment by mbercx** [comment]
Resolved in #301
